README.adoc

```diff
@@ -11,29 +11,29 @@
 //
 :page-layout: guide
 // INSTRUCTION: The project id is the part of the git repository after the guide- and must be specified
-:projectid: github repo name without the guide- prefix
+:projectid: outline
 // INSTRUCTION: Provide an estimate of how long the guide will take to go through.
 :page-duration: 15 minutes
 // INSTRUCTION: Provide the date when the guide is published.  Format is YYYY-MM-DD.
-:page-releasedate: 2017-11-17
+:page-releasedate: 2018-05-14
 // INSTRUCTION: Provide a description for the guide index page.
-:page-description: Learn how to create a REST service with JAX-RS, JSON-P, and Open Liberty.
+:page-description: Learn how to create a todo list API as a REST service using JAX-RS, CDI, and Open Liberty.
 // INSTUCTION: Please provide relevant tags, try to avoid inventing new ones, tags where there is 1 guide for the tag isn't useful.
-:page-tags: ['REST', 'Getting Started']
+:page-tags: ['REST', 'Todo']
 // INSTRUCTION: Specify the unique name of the guide that is used in the permalink.  For example below, it is rest-service
-:page-related-guides: ['']
+:page-related-guides: ['outline']
```

Comment:
This is incorrect. The related guides are existing guides that are determined to be related to the current guide.